docs/pages/jpscore/jpscore_inifile.md

```diff
@@ -6,267 +6,1219 @@ sidebar: jupedsim_sidebar
+#A, B: semi-axes of the ellipse
+#ANGLE: orientation of the ellipse
+#COLOR: color of the ellipse
+#V: speed of the pedestrian (in m/s)
+#Vx: x component of the pedestrian's velocity
+#Vy: y component of the pedestrian's velocity
+#FG: id of final goal
+#CG: id of current goal
+#Dx: x component of the pedestrian's desired direction
+#Dy: y component of the pedestrian's desired direction
+#SPOT: ped is highlighted
+#ROUTER: routing strategy used during simulation
+#GROUP: group of the pedestrian
+
+
+#ID	FR	X	Y	Z	A	B	ANGLE	COLOR	V	Vx	Vy	FG	CG	Dx	Dy	SPOT	ROUTER	GROUP
+1	0	55.70	103.00	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+2	0	52.70	102.10	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+3	0	54.80	100.60	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+4	0	53.60	102.70	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+5	0	50.30	102.70	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+6	0	54.20	100.30	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+7	0	55.40	102.10	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+8	0	52.70	100.60	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+9	0	53.00	103.30	0.00	0.15	0.15	0.00	0	0.00	0.00	0.00	0	16	0.00	0.00	0	2	1
+```
 
-This section defines constraints related to doors.
+## Optional Header Settings
+
+The following elements can be defined optionally in the header.
+
+- `<show_statistics>true</show_statistics>` Show different aggregate statistics e.g. the usage of the doors. (default:
+  false)
+- `<events_file>events.xml</events_file>` The name and location of the event file. Events can be used to open or close
+  doors at a certain point of time. For details see below.
+-
+- `<schedule_file>schedule.xml</schedule_file>`
+  The name and location of the schedule file. Schedules can be used to group doors and open or close this groups of door
+  at certain points of time. For details see below.
+
+### Events
+
+Events occur in time and trigger certain actions on doors and crossings. Following properties define an `event`:
+
+- `id` (int): unique id of the specific door (transition) as defined in the geometry file.
+  See [geometry](jpscore_geometry.html).
+- `time` (double): time of an event
+- `state` can be `close`, `temp_close`, `open` or `reset`: defines what event will be applied to the door
+
+Example of an event file:
 
 ```xml
- <traffic_constraints>
-    <doors>
-        <file>traffic.xml</file>
-    </doors>
- </traffic_constraints>
+ <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<JPScore project="JPS-Project" version="0.8">
+    <events update_frequency="2" update_radius="100" agents_color_by_knowledge="true">
+        <event time="30" state="close" id="5"/>
+        <event time="50" state="close" id="4"/>
+        <event time="70" state="open" id="4"/>
+        <event time="71" state="close" id="2"/>
+        <event time="80" state="open" id="5"/>
+    </events>
+</JPScore>
+```
+
+### Schedule
```

Comment:
Im not sure if anything from this section still works. Id remove it.

Reply:
Done